src/main/java/net/trollyloki/murdermystery/game/Game.java

```diff
@@ -513,8 +537,14 @@ public void run() {
             else
                 graceMessage = String.format(plugin.getConfigString("time.grace_warning"), this.graceTime);
         }
+        // Almost forgot to check if potatomode was on! If I hadn't caught that we'd be killing a null object!
+        if (isRunning() && hotpotatomode && this.potatoTime >= 0) {
+        	if (this.potatoTime == 0)
+        		kill(Bukkit.getPlayer(potatoVictim));
```

Comment:
What if the potato victim is dead or no longer online?

Reply:
I guess this will probably work, but you are relying on the kill code to set it to null if they leave, otherwise it will still break

Reply:
Yup, I am
If you have any better ideas feel free to change it, this is probably a lazy way anyways